projects/demo/src/pages/kit/date-range/examples/5-strict-mode/component.ts

```diff
@@ -0,0 +1,39 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {MaskitoDirective} from '@maskito/angular';
+import {TuiTextfieldControllerModule} from '@taiga-ui/core';
+import {TuiInputModule} from '@taiga-ui/kit';
+
+import mask from './mask';
+
+@Component({
+    standalone: true,
+    selector: 'date-range-mask-doc-example-5',
+    imports: [
+        TuiInputModule,
+        TuiTextfieldControllerModule,
+        FormsModule,
+        MaskitoDirective,
+    ],
+    template: `
+        <tui-input
```

Comment:
Could you add angular validator for valid date ?

To highlight invalid date (with changing value)
https://mui.com/x/react-date-pickers/adapters-locale/#with-luxon

Reply:
(the same comment for other examples about non-strict mode)

Reply:
https://github.com/taiga-family/maskito/issues/1221